saul-core/src/test/scala/edu/illinois/cs/cogcomp/saul/classifier/JoinTrainingTests/intializeSparseNetwork.scala

```diff
@@ -0,0 +1,86 @@
+package edu.illinois.cs.cogcomp.saul.classifier.JoinTrainingTests
+
+import edu.illinois.cs.cogcomp.infer.ilp.OJalgoHook
+import edu.illinois.cs.cogcomp.lbjava.infer.FirstOrderConstant
+import edu.illinois.cs.cogcomp.lbjava.learn.{ LinearThresholdUnit, SparseNetworkLearner }
+import edu.illinois.cs.cogcomp.saul.classifier.{ JointTrainSparseNetwork, ClassifierUtils, ConstrainedClassifier, Learnable }
+import edu.illinois.cs.cogcomp.saul.datamodel.DataModel
+import org.scalatest.{ FlatSpec, Matchers }
+
+/** Created by Parisa on 9/18/16.
+  */
+class initializeSparseNetwork extends FlatSpec with Matchers {
+
+  // Testing the original functions with real classifiers
+  "integration test" should "work" in {
+    // Initialize toy model
+    import testModel._
+    object TestClassifier extends Learnable(tokens) {
+      def label = testLabel
+      override def feature = using(word)
+      override lazy val classifier = new SparseNetworkLearner()
+    }
+    object TestBiClassifier extends Learnable(tokens) {
```

Comment:
`BiClassifier` stands for `BinaryClassifier`? If so, why not use the complete name?